SPECS/rust/rust.spec

```diff
@@ -3,13 +3,13 @@
 
 # Release date and version of stage 0 compiler can be found in "src/stage0.json" inside the extracted "Source0".
 # Look for "date:" and "rustc:".
-%define release_date 2023-02-09
-%define stage0_version 1.67.1
+%define release_date 2023-07-13
+%define stage0_version 1.71.0
 
 Summary:        Rust Programming Language
 Name:           rust
-Version:        1.68.2
-Release:        5%{?dist}
+Version:        1.72.0
```

Comment:
I think we should do a full from scratch build which includes `SPECS-EXTENDED`  w.r.t this change.